library/forms.inc

```diff
@@ -1,42 +1,36 @@
 <?php
 $GLOBALS['form_exit_url'] = "javascript:parent.closeTab(window.name, false)";
-
 function getFormById($id, $cols = "*")
 {
-    $sql = "select $cols from forms where id='$id' and deleted = 0 order by date DESC limit 0,1";
```

Comment:
Can't escape $cols since this is not a variable. So, just need to revert that part.